runtime/vm/ClassInitialization.cpp

```diff
@@ -481,6 +481,36 @@ classInitStateMachine(J9VMThread *currentThread, J9Class *clazz, J9ClassInitStat
 						}
 					}
 				}
+				
```

Comment:
I think we need to find out whether this check is done at classinit time or classloading time

Reply:
Classloading time might be too early. We need to handle the case where the `IdentityInterface` is indirectly implemented.  So we need all super classes (interfaces) be already loaded.

Reply:
Given our existing implementation i believe that is true. However, we should verify with RI that this is also the case.

Reply:
RI is doing it in the class loading time. We need to set a flag if a class cannot be super class of VT, and let its subclass inherit this flag.

Reply:
Change is updated to do this check earlier.